src/manifolds/Grassmann.jl

```diff
@@ -51,223 +49,235 @@ The manifold is named after
 
-Check whether `v` is a tangent vector in the tangent space of `x` on the [`Grassmann`](@ref)
-`M`, i.e. that `v` is of size and type as well as that
+Check whether `X` is a tangent vector in the tangent space of `p` on
+the [`Grassmann`](@ref) `M`, i.e. that `X` is of size and type as well as that
 
 ````math
-    x^{\mathrm{H}}v + v^{\mathrm{H}}x = 0_k,
+    p^{\mathrm{H}}X + X^{\mathrm{H}}p = 0_k,
 ````
 
 where $\cdot^{\mathrm{H}}$ denotes the complex conjugate transpose or Hermitian and $0_k$
-denotes the $k\times k$ zero natrix.
+denotes the $k \times k$ zero natrix.
 """
-function check_tangent_vector(G::Grassmann{n,k,F}, x, v; kwargs...) where {n,k,F}
-    t = check_manifold_point(G, x)
+function check_tangent_vector(G::Grassmann{n,k,F}, p, X; kwargs...) where {n,k,F}
+    t = check_manifold_point(G, p)
     t === nothing || return t
-    if (F === ℝ) && !(eltype(v) <: Real)
+    if (F === ℝ) && !(eltype(X) <: Real)
         return DomainError(
-            eltype(v),
-            "The matrix $(v) is not a real-valued matrix, so it can not be a tangent vector to the Grassmann manifold of dimension ($(n),$(k)).",
+            eltype(X),
+            "The matrix $(X) is not a real-valued matrix, so it can not be a tangent vector to the Grassmann manifold of dimension ($(n),$(k)).",
         )
     end
-    if (F === ℂ) && !(eltype(v) <: Real) && !(eltype(v) <: Complex)
+    if (F === ℂ) && !(eltype(X) <: Real) && !(eltype(X) <: Complex)
         return DomainError(
-            eltype(v),
-            "The matrix $(v) is neiter real- nor complex-valued matrix, so it can not bea tangent vector to the complex Grassmann manifold of dimension ($(n),$(k)).",
+            eltype(X),
+            "The matrix $(X) is neiter real- nor complex-valued matrix, so it can not bea tangent vector to the complex Grassmann manifold of dimension ($(n),$(k)).",
         )
     end
-    if size(v) != representation_size(G)
+    if size(X) != representation_size(G)
         return DomainError(
-            size(v),
-            "The matrix $(v) is does not lie in the tangent space of $(x) on the Grassmann manifold of dimension ($(n),$(k)), since its dimensions are wrong.",
+            size(X),
+            "The matrix $(X) is does not lie in the tangent space of $(p) on the Grassmann manifold of dimension ($(n),$(k)), since its dimensions are wrong.",
         )
     end
-    if !isapprox(x' * v + v' * x, zeros(k, k); kwargs...)
+    if !isapprox(p' * X + X' * p, zeros(k, k); kwargs...)
         return DomainError(
-            norm(x' * v + v' * x),
-            "The matrix $(v) is does not lie in the tangent space of $(x) on the Grassmann manifold of dimension ($(n),$(k)), since x'v + v'x is not the zero matrix.",
+            norm(p' * X + X' * p),
+            "The matrix $(X) is does not lie in the tangent space of $(p) on the Grassmann manifold of dimension ($(n),$(k)), since x'v + v'x is not the zero matrix.",
```

Comment:
```suggestion
            "The matrix $(X) does not lie in the tangent space of $(p) on the Grassmann manifold of dimension ($(n),$(k)), since x'v + v'x is not the zero matrix.",
```